### DISCUSSION TITLE
open two docusaurus tabs in chrome, the webpage crashes

### DISCUSSION THREAD

**Discussion by WLyKan** [facebook/docusaurus#7700] [Q&A]
when i open two docusaurus tab in chrome (Version 103.0.5060.53), the tab crush

![image](https://user-images.githubusercontent.com/2155093/176386536-0dfccce5-cb52-490a-b61a-fbfbde1b6e6e.png)

**Comment by Josh-Cena** [comment]
@WLyKan Please provide a reproduction.

**Reply by WLyKan** [reply]
just open more than one docusaurus tab in chrome

**Comment by WLyKan** [comment]
in Microsoft Edge (Version 102.0.1245.44 )，it did not happen. May be it is a bug in Version 103

**Reply by WLyKan** [reply]
I also tried in Chrome  Canary (Version 105.0.5148.0), it did not happen too

**Comment by Josh-Cena** [comment]
Does it reliably reproduce on Edge 103? Does it reproduce on other Docusaurus sites, e.g. https://typescript-eslint.io/?

**Reply by WLyKan** [reply]
can not reproduce on Edge 103, just happen in my chrome (Version 103.0.5060.53) normal window, and did not happen in incognito window.

**Comment by Josh-Cena** [comment]
I can't reproduce this on Chrome 103.0.5060.53. Two tabs open and it's still working as usual 🤔

- If you clear cache for the site, does it work?
- What's your OS?
- Do other sites like https://typescript-eslint.io/ work?

**Comment by slorber** [comment]
It's impossible to help with this kind of issue, please do an effort and help us help you.

Please do:
- tell us if it's prod or dev mode
- if it's prod, provide the url
- if it's dev, give us a repro
- test on other open-source Docusaurus sites and if it only affects your site or others too
- show us console logs when it crashes
- ensure you don't have chrome plugins: use a new browser session without any plugin
- ensure your windows session does not have anything weird that could mess up with anything. Create a new one to test if needed. Disable your fancy antivirus/firewall.
